### PR TITLE
Bug 1274328 - Make it possible to hide Windows 7 VM 32-bit tests

### PR DESCRIPTION
**Pull request by camd**
Split Windows & tests to VM and non-VM based on the buildername

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1561)

### CODE REVIEW

File: treeherder/etl/buildbot.py [hunk 1]
```diff
@@ -231,6 +231,14 @@
         }
     },
     {
+        'regex': re.compile(r'WINNT 6\.1|win7|Windows 7 VM.*32-bit', re.IGNORECASE),
```
Comment (edmorley) [inline treeherder/etl/buildbot.py]: The earlier parts of the regex duplicate the non VM entry?

File: ui/js/values.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ treeherder.value("thPlatformNameMap", {
     "osx-10-11": "OS X 10.11",
     "windowsxp": "Windows XP",
     "windows7-32": "Windows 7",
+    "windows7-32 VM": "Windows 7 VM",
```
Comment (edmorley) [inline ui/js/values.js]: How about 'windows7-32-vm' for consistency with the other entry's whitespace and capitalisation?

File: treeherder/etl/buildbot.py (hunk 1 above)
Comment (edmorley) [inline treeherder/etl/buildbot.py]: Eg the WINNT and win7 bits will match both the VM and non-VM entries, and which ends up matching will just be based on order

File: ui/js/values.js (hunk 2 above)
Comment (camd) [inline ui/js/values.js]: sheesh, you're totally right...  :)  fixing...

**Comment by camd** [comment]
I did some queries on stage for buildernames and I found these three formulations wrt "Windows 7%32-bit%":

Windows 7 32-bit try talos xperf-e10s
Windows 7 VM 32-bit ash debug test cppunit
Windows 7 VM-GFX 32-bit try opt test xpcshell

All 3 of these are covered by unit tests and now the regex is not order dependent.

**Pull request merged**